data/Models/Friend.js

```diff
@@ -17,3 +17,39 @@ exports.getListOfFriends = () => {
     });
   });
 };
+
+export getFriendById = (id) => {
+  return new
+};
+
+exports.addFriend = ({id, name, email, wish, hash}) => {
+  const friend = new FriendModel();
+  friend.id = id;
+  friend.name = name;
+  friend.email = email;
+  friend.wish = wish;
+  friend.hash = hash;
+  return new Promise((resolve, reject) => {
+    friend.save((err, f) => {
+      if (err) reject(err);
+      console.log('Friend saved!', friend);
+      resolve(f._id);
```

Comment:
This was for mongoose (to associate the friend with the group - via the mongoose `._id`).